src/ui/server/controllers/insurance/business/company-details/index.ts

```diff
@@ -121,6 +124,12 @@ const postCompaniesHouseSearch = async (req: Request, res: Response) => {
   }
 };
 
+const redirectToNoCompaniesHouseNumberExitPage = (req: Request, res: Response) => {
+  req.flash('exitReason', exitReasonNoCompaniesHouseNum);
```

Comment:
What do you think about doing this instead of having `exitReasonNoCompaniesHouseNum`?

```js
req.flash('exitReason', PAGES.INSURANCE.APPLY_OFFLINE.REASON.NO_COMPANIES_HOUSE_NUMBER);
```

Or maybe just rename `exitReasonNoCompaniesHouseNum` to `exitReason`?